system/include/platform.h

```diff
@@ -77,6 +77,15 @@
 #define _GNU_SOURCE
 #endif
 #include <endian.h>
+
+// Used for python 2.7 builds on cecntos 6
+//#ifdef _POSIX_C_SOURCE
+//#undef _POSIX_C_SOURCE
+//#endif
+//#ifdef _XOPEN_SOURCE
+//#undef _XOPEN_SOURCE
+//#endif
+ 
```

Comment:
Should this really be for a certain version of Python and perhaps the compiler ?
(I understand that the OS often dictates the default version of Python and compiler but to be accurate should the comment really be for a specific version of Python ... ?
Could this logic/check/comment go into a pyembed.cpp file instead ?

Reply:
I'll look into moving it into the pyembed code, that's a great idea.